src/client/commands/jobs.rs

```diff
@@ -62,28 +75,33 @@ pub async fn output_job_detail(
 pub async fn cancel_job(
     _gsettings: &GlobalSettings,
     connection: &mut ClientConnection,
-    job_id: JobId,
+    job_ids: Vec<JobId>,
 ) -> crate::Result<()> {
-    let response =
-        rpc_call!(connection, FromClientMessage::Cancel(CancelRequest { job_id }), ToClientMessage::CancelJobResponse(r) => r).await?;
+    let mut sorted_ids = job_ids.clone();
```

Comment:
Why do we need to sort the IDs? If we do, I think that there's no need to clone them.